.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Run Smoke Tests'
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+  workflow_dispatch:
```

Comment:
@EvanPurkhiser With one of our bots, we can hit an API that allows triggering this.

As @BYK mentioned, we can listen to these kind of events and decide what we want to do with it.